src/hb-ot-cff1-table.hh

```diff
@@ -1138,8 +1137,7 @@ struct cff1
 	if (unlikely (!priv_interp.interpret (*priv))) { fini (); return; }
 
 	priv->localSubrs = &StructAtOffsetOrNull<CFF1Subrs> (&privDictStr, priv->subrsOffset);
-	if (priv->localSubrs != &Null(CFF1Subrs) &&
-	    unlikely (!priv->localSubrs->sanitize (&sc)))
+	if (priv->subrsOffset && !priv->localSubrs->sanitize (&sc))
```

Comment:
Apparently CFF1Subrs::sanitize (and CFF2Subrs) isn't returning true on NullPool objects which isn't good.

Reply:
`Number of object data. Note there are (count+1) offsets` hmm, so we can't do anything about